XYZDrivers/src/main/webapp/admin.jsp

```diff
@@ -9,23 +9,7 @@
         <link rel="stylesheet" type="text/css" href="styles/main.css" />
     </head>
     <body>
-        <nav class="navbar navbar-default">
-            <div class="container-fluid">
-              <div class="navbar-header">
-                <a class="navbar-brand" href="#">XYZ Drivers Association</a>
-              </div>
-
-              <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <ul class="nav navbar-nav">
-                  <li><a href="#">Home</a></li>
-                </ul>
-  
-                <ul class="nav navbar-nav navbar-right">
-                  <li><a href="#">Logout</a></li>
-                </ul>
-              </div><!-- /.navbar-collapse -->
-            </div><!-- /.container-fluid -->
-        </nav>
+        <jsp:include page="nav.jsp"></jsp:include>
```

Comment:
Make sure to explain what partials are to the group monday in case someone is asked about them from mehmet

Reply:
Good call